presentation/src/main/java/com/example/healthysalad/MainActivity.kt

```diff
@@ -2,17 +2,16 @@ package com.example.healthysalad
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import com.example.data.Data
-import com.example.domain.Base
-import com.example.domain.Order
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        val order = Order()
+
+        var titleFragmentInstance : TitleFragment = TitleFragment.newInstance()
```

Comment:
everybody knows that this line create `TitleFragment instance`, so good name is a just `titleFragment`.